src/AppBundle/Controller/BlogController.php

```diff
@@ -150,4 +151,31 @@ public function commentFormAction(Post $post)
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/search", name="blog_search")
+     * @Method("GET")
+     *
+     * @return JsonResponse
```

Comment:
@yceruto, the controller returns a `Response` instance when a request is not an `XMLHttpRequest`, so I think `JsonResponse` is not enough for this case.